torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -359,6 +360,19 @@ def inner(*flat_args):
                 ]
             )
 
+            # Compute the offset of the ViewMeta sequence of the output.
+            # Consider that the output 'O' aliases one of the inputs 'I':
```

Comment:
hmm - I might be missing something but it's not clear to me why we need the extra complexity of having to worry about whether `I` (the graph input) already has some `ViewMetas` on it.

If one of our inputs to the graph is an existing alias tensor (e.g. it was created by taking a `view(..)` off of some other tensor from eager mode), then:

(1) When AOTAutograd runs functionalization, it will convert the input to a FakeTensor, then wrap that FakeTensor into a `PythonFunctionalTensor(C++FunctionalTensorWrapper(FakeTensor))`

(2) The generated `FunctionalTensorWrapper` will **not** stash any of the previous `ViewMetas` that came from eager mode - it will treat the current input as having an empty view chain.

The main reason for this is that (a) it's not really necessary, and (b) we have no guarantee that we can even get these ViewMetas that were created in eager mode. Autograd sometimes stores them, but in the common case does not actually store these view chains in eager mode.

So given that, do we ever actually want to regenerate only a subset of the base's view chain? Or is it sufficient to always re-run the **entire** view chain (in which case we don't need the extra functionality of a start / end index).

Reply:
Oh - reading your PR description again, I guess this happens whenever an input gets an inplace_view mutation.

That makes sense. Can you mention that in the comment?

Reply:
Explicitly not supporting in-place view operations, so marking as resolved.